src/ostree/ot-admin-builtin-deploy.c

```diff
@@ -29,7 +29,7 @@
 #include "ostree.h"
 #include "otutil.h"
 
-#include "../libostree/ostree-kernel-args.h"
+#include "ostree-kernel-args.h"
```

Comment:
Minor nit: can also remove this include and have it pulled in by `"ostree.h"`

Reply:
Right.. Will be updating :+1: